doc/dev/MAINTAINERS_AND_CONTRIBUTORS.md

```diff
@@ -20,6 +20,7 @@ Contributors with a single contribution or that may not want to are not listed h
 | **Name/Alias** | **Handle** | **Active** |
 | -------------- | ---------- | ---------- |
 | jrushlow       | 19jrushlow | Yes        |
+| samoncrief     | samoncrief | Yes        |
```

Comment:
Hi @samoncrief 

Do you think you could synchronise your alias between F3D discord and here ? It can be one way or another.